src/app/components/common/add-user-dialogue/add-user-dialogue.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup} from '@angular/forms';
+import { MatDialogRef } from '@angular/material/dialog';
+import { UserService } from '@app/services/user/user.service';
+
+@Component({
+  selector: 'app-add-user-dialogue',
+  templateUrl: './add-user-dialogue.component.html',
+  styleUrls: ['./add-user-dialogue.component.scss']
+})
+export class AddUserDialogueComponent implements OnInit {
+  // Angular forms system works with a FormGroup
+  form: FormGroup;
+
+   constructor(
+    private userService : UserService,
+    private formBuilder: FormBuilder,
+    public dialogRef: MatDialogRef<AddUserDialogueComponent>
+  ) {}
+
+  ngOnInit(): void {
```

Comment:
Please change this to use form builder instead of form groups and controls. Refer to the login component.